src/dependencies/FileDownloader.ts

```diff
@@ -58,3 +44,9 @@ export class FileDownloader implements DependencyInstaller {
 		}
 	}
 }
+
+interface GotProgress {
+	percent: number;
+	transferred: number;
+	total: number;
+}
```

Comment:
It looks like you could just be getting this from the `@types/got` package, rather than defining it yourself ([relevant section](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7bcd046f24df925802e1d1d04a7fd6f19b54cde0/types/got/index.d.ts#L377-L381)).

Reply:
Good catch! I was looking for that type definition but couldn't find it myself